lib/invisible_captcha/controller_ext.rb

```diff
@@ -98,8 +98,8 @@ def honeypot_spam?(options = {})
         end
       else
         InvisibleCaptcha.honeypots.each do |default_honeypot|
-          if params[default_honeypot].present?
-            warn_spam("Invisible Captcha honeypot param '#{default_honeypot}' was present.")
+          if params[default_honeypot].present? || params.dig(scope, default_honeypot).present?
+            warn_spam("Invisible Captcha honeypot param '#{scope}.#{default_honeypot}' was present.")
```

Comment:
Maybe we should remove this `#{scope}.` prefix? For cases with no scope, it will be a bit weird (we neither print it in the `if honeypot` branch, which makes it consistent)

Reply:
I strongly feel that having some indication of the honeypot being scoped would be good to have in the message. I'm very open to hearing other solutions that may work better than my first attempt 🙂

Reply:
Ok! let's go with that for now. If we come up with a better idea, we can change it later!